source/likely.js

```diff
@@ -2,5 +2,6 @@
 
 var likely = require('./index.js');
 
-window.likely = likely;
 window.addEventListener('load', likely.initiate);
```

Comment:
I'm afraid I have no free time this week to implement this feature, but I can implement it later, if it's not critical.